bases/rsptx/interactives/runestone/activecode/js/activecode.js

```diff
@@ -197,9 +200,12 @@ export class ActiveCode extends RunestoneBase {
             this.codeCoachList.push(new PyflakesCoach());
         }
 
+        // Why is this necessary???
```

Comment:
It was added in 2020.... I don't remember why, there was probably some kind of timing issue that this fixed.  May or may not even exist anymore.  Did you try removing it?

Reply:
I tried turning it off and didn't notice any issues. But I wasn't confident there isn't something out there relying on it.